ocean_lib/data_provider/data_service_provider.py

```diff
@@ -68,6 +68,34 @@ def encrypt_files_dict(files_dict, encrypt_endpoint, asset_id, publisher_address
 
             return response.json()['encryptedDocument']
 
+    @staticmethod
+    def check_url(url, check_url_endpoint):
+        payload = json.dumps({'url': url})
+
+        response = DataServiceProvider._http_client.post(
+            check_url_endpoint, data=payload,
+            headers={'content-type': 'application/json'}
+        )
+
+        if (
+            not response or
+            not hasattr(response, 'status_code') or
+            response.status_code != 200
+        ):
+            msg = (f'Could not determine Content-Length and Content-Type '
+                   f'{check_url_endpoint}, status {response.status_code}')
+            logger.error(msg)
+            raise OceanEncryptAssetUrlsError(msg)
+
+        result = response.json()['result']
+
+        logger.info(
+            f"Check URL was successful, content type: {result['contentType']},"
+            f" content length {result['contentLength']}')"
+        )
+
+        return result
```

Comment:
what is the type of `result`, is it a text message or simply a boolean

Reply:
it is currently a json, but we can return a boolean instead, for simplicity. Let me know how I should proceed.

Reply:
CC @ssallam: @alexcos20 said it must be a dictionary, not a boolean

Reply:
the response should be :
{  valid = false,  result={"contentLength":"", "contentType":""} }